ts/DataGrid/Table/Header/HeaderCell.ts

```diff
@@ -135,7 +135,7 @@ class HeaderCell extends Cell {
 
         // Render content of th element
         this.row.htmlElement.appendChild(this.htmlElement);
-        this.headerContent = makeHTMLElement('div', {
+        this.headerContent = makeHTMLElement('span', {
```

Comment:
Why changing `div` to `span`? Display stays the same 🤔

Reply:
I wanted the voiceover to treat it as an inline element, even though display is still `block` (forced by css).